modern/src/utils.js

```diff
@@ -92,12 +97,11 @@ export async function inlineIncludes(xml, zip) {
       return node;
     }
     // TODO: Normalize file names so that they hit the same cache
-    const includedFile = await readXml(zip, node.attributes.file);
+    const filename = fixFilenameSlashes(node.attributes.file);
+    const includedFile = await readXml(zip, filename);
```

Comment:
I think this would lead to us double escaping.

Reply:
Oh derp, missed that `readXml` already calls `getCaseInsensitveFile`. Won't double escape, since we are just replacing, but doesn't need to be in both places!

Thanks